project_stage_state/views/project_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
```

Comment:
A license and copyright notice would be great over here as well, though not required

Reply:
@alexey-pelykh , Could you please check it now?